### DISCUSSION TITLE
Getting midi from arrange into pt

### DISCUSSION THREAD

**Discussion by jimmydeer** [helgoboss/helgobox#1130] [Q&A] [answered]
I have a song in playtime that is a long phrase, so recording waiting for it to start every time is long (I know pre-roll is coming). So I just recorded it in the arrange, thinking I could drag it in, but I can't, and copy/paste doesn't work either. Is there a way to do it now, or just manually record?

**Comment by helgoboss** [comment, marked as answer]
Right-click the slot and choose "Import selected item into slot".

**Comment by jimmydeer** [comment]
Sweet thx



Kind regards,

*jimmy deer*
jimmydeer.com

  <https://www.instagram.com/jimmydeermusic/>
<https://www.facebook.com/JimmyDeerMusic/>
